.gitignore

```
@@ -1 +1 @@
-node_modules
+zinnia
```

Should we ignore `zinnia.exe` as well, to support developers on Windows?

```suggestion
zinnia
zinnia.exe
```

---

Yes, we should build inclusive community :)